pkg/api/types.go

```diff
@@ -504,20 +505,31 @@ func (l *VolumeList) Set(value string) error {
 	if len(value) == 0 {
 		return errors.New("invalid format, must be source:destination")
 	}
+
+	var pos int
+	isWindows, _ := regexp.MatchString("^[A-Z]:\\.*", value)
```

Comment:
yes.  i mean it's basically case insensitive, so users can type "c:\foo" which maps to C:\foo effectively.